dangerfile.js

```diff
@@ -32,3 +33,8 @@ danger.git.created_files.filter((filepath) => filepath.includes('.changeset/') &
             console.log(err);
         });
     });
+
+// Check for empty PR Description checkboxes
+if (pr.body.includes('- [ ]')) {
```

Comment:
Do you need to specifically check for the `x` here as i think you could enter anything and this would pass right?